### ISSUE TITLE
[MAS2.1.1][Keyboard-Live chat dialog] "Root" list box present in "JSON" pane are not expanded or collapsed by keyboard.

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1914]
**Actual Result:**
When we click "Root" list box it will not expanded or collapsed whereas same can be accessible by mouse.

**Expected Result:**
It should expanded or collapsed as it can be done by mouse.

**Note:**
1. Same issue observed on the Linux system. Please refer "JSON pane not expanded collapsed by key_Linux" attachment.
2. Same issue observed for "Conversation", "From" and "Recipient" list box available under "Root" list box.

**Observation:**
1. We already try to activate it with various shortcut keys but not be able to activate the same.
2. In Linux system the pane is expanded by left/right arrow.

**Repro Steps:**
1. Open the Bot Framework V4 Emulator.
2. Welcome tab screen gets opened.
3. Navigate to the "Create new bot configuration" button and activate it.
4. "Create new bot configuration" dialog box gets opened.
5. Fill the required fields like bot name and endpoint URL example: http://localhost:3978/api/messages
6. Navigate to "Save and Connect" button and activate it.
7. Now, navigate through the pane and observe the issue after "Save and Connect" button.
8. "Live Chat" screen will appear.
9. Navigate to "[ConversationUpdate event detected]" button and activate it.
10. Observe the change in "JSON" tab and navigate to the tab.
11. Try to collapse "Root" list box by keyboard.
12. Observe the issue.

**User Impact:**
The keyboard dependent user will get confused as they will not be able to knew the presence of content or controls in the "Root" or other list box.

**Test Environment:**
OS: MacOs Version 10.15
Application: Bot Framework V4 Emulator (Version 4.5.2-80829)

OS: Ubuntu 18.04.3 LTS
Application: BotFramework-Emulator-4.5.2-linux

**Impacted MAS Link:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}](url)

**Attachment:**
[not expanded or collapsed by keyboard.zip](https://github.com/microsoft/BotFramework-Emulator/files/3696197/not.expanded.or.collapsed.by.keyboard.zip)
[JSON pane not expanded collapsed by key_Linux.zip](https://github.com/microsoft/BotFramework-Emulator/files/3819168/JSON.pane.not.expanded.collapsed.by.key_Linux.zip)

**Comment by tonyanziano** [comment]
@Kumar2608 @awalia13 @ShikhaMishra11 

Currently, the user should be able to expand / collapse JSON nodes in the tree view using the right or left arrow keys. Are these not appropriate key binds for an accessible tree view component? If not, what keys should be used to expand and collapse the nodes. (Space? Enter?)

**Comment by ShikhaMishra11** [comment]
Yes currently tree view is accessible with right or left arrow keys which is appropriate key to accessible tree view so we can close the issue if you suggest.

**Comment by tonyanziano** [comment]
Closing due to above comment.

**Comment by ShikhaMishra11** [comment]
As per above comment we are closing the bug
